docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,187 @@
+```
+
+That means the node has detected the subnet, and is attempting to initialize it and start bootstrapping the Subnet. It might take some time (if there are already transactions on the subnet), and eventually it will finish the bootstrap with a message like:
+
+```text
+Jul 30 18:27:21 node-fuji avalanchego[1728308]: [07-30|18:27:21.055] INFO <2ebCneCbwthjQ1rYT41nhd7M76Hc6YmosMAQrTFhBq8qeqh6tt Chain> snowman/transitive.go:333 consensus starting with J5wjmotMCrM2DKxeBTBPfwgCPpvsjtuqWNozLog2TomTjSuGK as the last accepted block
+```
+
+That means the node has successfully bootstrapped the Subnet and is now in sync. If the node is one of the validators, it will start validating any transactions that get posted to the Subnet.
+
+### Monitoring
+
+If you want to inspect the process of Subnet syncing, you can use the RPC call to check for the [blockchain status](../apis/avalanchego/apis/p-chain.md#platformgetblockchainstatus).
+
+For a more in-depth look into Subnet operation, check out the blockchain log. By default, the log can be found in `~/.avalanchego/logs/ChainID.log` where you replace the `ChainID` with the actual ID of the blockchain in your Subnet.
+
+For an even more thorough (and pretty!) insight into how the node and the Subnet is behaving, you can install the Prometheus+Grafana monitoring system with the custom dashboards for the regular node operation, as well as a dedicated dashboard for Subnet data. Check out the [tutorial](../nodes/maintain/setting-up-node-monitoring.md) for information on how to set it up.
+
+### Managing validation
+
+On Avalanche all validations are limited in time and can range from two weeks up to one year. Furthermore, subnet validations are always a subset of the Primary Network validation period (must be shorter or the same). That means that periodically your validators will expire and you will need to submit a new validation transaction for both the Primary Network and your Subnet.
+
+Unless managed properly and in a timely manner, that can be disruptive for your Subnet (if all validators expire at the same time your Subnet will halt). To avoid that, keep notes on when a particular validation is set to expire and be ready to renew it as soon as possible. Also, when initially setting up the nodes, make sure to stagger the validator expiry so they don't all expire on the same date. Setting end dates at least a day apart is a good practice, as well as setting reminders for each expiry.
+
+## Conclusion
+
+Hopefully, by reading this document you have a better picture of the requirements and considerations you need to make when deploying your subnet to production and you are now better prepared to launch your Subnet successfully.
+
+Keep in mind, running a Subnet in production is not a one-and-done kind of situation, it is in fact running a fleet of servers 24/7. Things can change and go sideways in unexpected ways so you always need to be on your toes, and constantly monitor the nodes and subnet health. 
```

Comment:
Can we change the gist of this from "this could break in any way" to, this is a running fleet of servers that requires constant monitoring as with any critical real time service?

Reply:
Will do.